README.md

```diff
@@ -687,6 +687,7 @@ oneview_power_device '<iPDU hostname>' do
   client <my_client>
   username <username>
   password <password>
+  auto_import_cert [true, false] # Optional. Only used with the :discover action. Default value is false.
```

Comment:
Suggestion: Maybe it was good that the name was ```import_certificate```. It is more readable.

Reply:
I do like `import_certificate`, but I feel it is not clear that is doing so automatically. 
Like, in the case of `import_certificate` I'd expect a field to pass in the certificate.
Perhaps `auto_import_certificate` ? 
My naming sense sucks though, If you guys don't feel like it lacks clarity `import_certificate` is also fine by me.

Reply:
I'll change the name to `auto_import_certificate`, right?! =D \o/